packages/toolpad-app/src/components/HarViewer.tsx

```diff
@@ -11,6 +11,22 @@ function fixLinks(elm: Element) {
   elm.querySelectorAll('a').forEach((link) => link.setAttribute('target', '_blank'));
 }
 
+function forceDarkMode(elm: Element) {
+  elm.querySelectorAll('svg').forEach((svg) => {
+    svg.setAttribute('fill', 'white');
+    const fullLabel = svg.getElementsByClassName('rect.label-full-bg');
+    if (fullLabel && fullLabel.length > 0) {
+      (fullLabel[0] as SVGElement).style.fill = '#5090D3'; // theme.palette.primary.main
```

Comment:
I'm curious, why not import the theme? or at the very least `darkBlue[500]`?

Reply:
Agreed, this is how it should have been done. Fixed